### PR TITLE
Chore/release deb

### PR DESCRIPTION
**Pull request by da13da**
We have added a feature to release .deb packages using GitHub Actions. 
The functionality has been verified using act for parts that do not rely on GitHub.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -137,13 +165,22 @@ jobs:
...
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Upload asset (mac)
         uses: ./.github/actions/upload-release-asset
         with:
           release_id: ${{ steps.release.outputs.result }}
           asset_path: ${{ github.workspace }}/nodex-agent-aarch64-apple-darwin.zip
           asset_name: nodex-agent-mac.zip
-          asset_content_type: application/zip
+          asset_content_type: application/vnd.debian.binary-package
```
Comment (curry-like) [inline .github/workflows/release.yml:183]: Is this correct?

Comment (da13da) [reply .github/workflows/release.yml]: Thank you. I was specifying the wrong location. I've fixed it.

**Comment by da13da** [comment]
@curry-like 
I would appreciate it if you could wait for a review for a while so that I can modify it to compile with cross.

**Comment by da13da** [comment]
When I do a cross build with act, the health check fails, although I don't know the cause.
So I checked with github action.
https://github.com/nodecross/nodex/actions/runs/10645001794/job/29510340783?pr=375

**Review by curry-like — APPROVED** [review]

**Pull request merged**